src/utils/isValidTwitterSignature.ts

```diff
@@ -0,0 +1,45 @@
+import { hexToBytes } from '.';
```

Comment:
what about the name of the file `TwitterSignature` or `TwitterSignatureValudator`?